config/test/config.js

```diff
@@ -63,7 +63,7 @@ module.exports = {
 	},
 	js_debug_level: 'debug',
 	logfile: {
-		silent: true,
+		silent: false,
```

Comment:
need to set this back to true otherwise it's very difficult to read the travis log when things to wrong

If you need it to be set to false then we should probably make a copy of this config

Reply:
set back